src/services/helper/entity.js

```diff
@@ -65,6 +67,16 @@ module.exports = class EntityHelper {
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
+			console.log(_id)
+			let key = ''
+			if (bodyData.type) {
+				key = 'mentoring_entity_' + bodyData.type
+				await InternalCache.delKey(key)
+			} else {
+				const entities = await entitiesData.findOne(_id)
+				key = 'mentoring_entity_' + entities.type
+				await InternalCache.delKey(key)
```

Comment:
@rakeshSgr we then have to write list function query again in update and create to avoid that I am deleting key and when list API is called it will automatically cache the data.